lib/Runtime/Language/SourceTextModuleRecord.cpp

```diff
@@ -587,8 +594,12 @@ namespace Js
             });
             if (FAILED(hr))
             {
-                JavascriptError *error = scriptContext->GetLibrary()->CreateError();
-                JavascriptError::SetErrorMessageProperties(error, hr, _u("fetch import module failed"), scriptContext);
+                if (error == nullptr)
+                {
```

Comment:
I want to propagate the inner error outward, so I only create the "fetch import module failed" error if we don't have a better one to use.